### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aoxolotl**
Addresses Issue #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
 # lagoon
 [![Build Status](https://travis-ci.org/combatopera/lagoon.svg?branch=master)](https://travis-ci.org/combatopera/lagoon)
+
+`lagoon` is a library for importing programs by name as if they were functions
+
+## Install
+`lagoon` has a dependency on GNU screen which can be installed using apt
+```
+$ sudo apt-get install screen
+```
```
Comment (combatopera) [inline README.md:10]: this is misleading - only the screen.py module requires gnu screen, it's not needed to be able to use the rest of lagoon. would it make more sense to have this message in the Concern readme, as that does not work without gnu screen? ideally users shouldn't have to dig into the readmes of libraries (lagoon) to be able to use the application (Concern)

**Review by combatopera — CHANGES_REQUESTED** [review]
thanks for the pr, please see inline comment

**Comment by combatopera** [comment]
hi @aoxolotl, are you still working on this?

**Comment by aoxolotl** [comment]
Hey! I will wrap it up this weekend

File: README.md (hunk 1 above)
Comment (aoxolotl) [reply README.md:10]: Cool. So I'll shift this to Concern readme and say "screen.py module requires gnu screen which can be installed using..."

**Pull request closed without merging**